### PR TITLE
Cura 5676 add contacts toolbox

### PR DESCRIPTION
**Pull request by alekseisasin**
Change contact fields in Toolbox

### CODE REVIEW

**Comment by fieldOfView** [comment]
Aww, you still had my branch merged in...
If you merge this PR like this, merging mine will fail.

File: plugins/Toolbox/resources/qml/ToolboxAuthorPage.qml [hunk 1]
```diff
@@ -114,7 +114,7 @@ Item
                 {
                     if (details.website)
                     {
-                        return "<a href=\""+details.website+"\">Link</a>"
+                        return "<a href=\""+details.website+"\">"+details.website+"</a>"
```
Comment (fieldOfView) [inline plugins/Toolbox/resources/qml/ToolboxAuthorPage.qml]: code style: `" + details.website + "`

Comment (LipuFei) [reply plugins/Toolbox/resources/qml/ToolboxAuthorPage.qml]: Fixed in 98d8a3f

**Comment by alekseisasin** [comment]
@fieldOfView , sorry for this I made a new branch from your PR branch,

**Comment by fieldOfView** [comment]
No worries, it seems to have worked itself out because my setting is still in master. Github black magic.

**Pull request merged**